dkron/api.go

```diff
@@ -357,14 +367,63 @@ func (h *HTTPTransport) executionsHandler(c *gin.Context) {
 		return
 	}
 
+	apiExecutions := make([]*apiExecution, len(executions))
+	for j, execution := range executions {
+		apiExecutions[j] = &apiExecution{execution, false}
+		if outputSizeLimit > -1 {
+			// truncate execution output
+			if outputSizeLimit == 0 {
+				apiExecutions[j].Output = ""
+				apiExecutions[j].OutputTruncated = true
```

Comment:
When len(execution.Output) == 0 && outputSizeLimit == 0, OutputTruncated should be false.
Is this special case for outputSizeLimit==0 particularly useful? Seems like it would work fine with just the bit currently in the else block.

Reply:
you're right, not exactly sure why I added that condition. might have been related to the initial "hacky" implementation..
or maybe it was trying to save a `len()` operation whenever output is not wanted at all